examples/window.rs

```diff
@@ -1069,6 +1128,7 @@ const KEY_BINDINGS: &[Binding<&'static str>] = &[
     Binding::new("T", ModifiersState::SUPER, Action::CreateNewTab),
     #[cfg(macos_platform)]
     Binding::new("O", ModifiersState::CONTROL, Action::CycleOptionAsAlt),
+    Binding::new("S", ModifiersState::empty(), Action::Message),
```

Comment:
You don't need to bind it since it does nothing.

Reply:
It does, see https://github.com/rust-windowing/winit/pull/3801#discussion_r1688280503.